### DISCUSSION TITLE
I can't upload JPEG files to Lychee.

### DISCUSSION THREAD

**Discussion by Mowen-M** [LycheeOrg/Lychee#2713] [Bugs]
### Lychee version

V6.0.1

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

I can't upload JPEG files to Lychee. I tried many machines whatever pc or phones it all break down. PNG ,JPG is OK.

### Steps to reproduce the issue

1. using PC or phones
2. Upload JPEG files
3. it will 0-100% but while a time after it crashed.
4. something weng wrong see log
5. log is rufused url. But others Image is OK for upload

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Mowen-M** [comment]
By the way, In the docker log, it said http 499 /0. It maybe the problem of internet ( i tried it in local internet, and still got error), The issues about time out by need more time to connect.

**Reply by ildyria** [reply]
> log is rufused url. But others Image is OK for upload

it seems that you did not configure your `APP_URL` correctly.

We really need to see your diagnostics. You seem to have access to your app, so the Diagnostic page is 100% working.

You mention docker but it is unclear to us if you are using your own docker image, if you are using the official LycheeOrg docker image or the LinuxServer one.

**Reply by Mowen-M** [reply]
> official LycheeOrg docker image

official LycheeOrg docker image，and url set correct and turn on https force

**Comment by Mowen-M** [comment]
![image](https://github.com/user-attachments/assets/8f14213f-70a0-472a-8eb0-11b51dc4d7bc)

**Comment by Mowen-M** [comment]
![image](https://github.com/user-attachments/assets/89db4265-e3d9-4eef-8f1f-f75ce6ad8d54)

**Comment by Mowen-M** [comment]
![IMG_5879 - Copy](https://github.com/user-attachments/assets/4f77ed95-ce49-45e9-af58-48477ae066e3)
